### PR TITLE
New client, typescript

### PR DESCRIPTION
**Pull request by smithamax**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,29 +1,53 @@
...
     "express": "^4.15.2",
     "pify": "^2.3.0",
     "prettier": "^2.3.0",
-    "prom-client": "^12.0.0"
+    "prom-client": "^12.0.0",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.8.2"
   },
   "dependencies": {
+    "@types/node-fetch": "^2.6.2",
```
Comment (denwilliams) [inline package.json]: Should be devDeps?

File: package.json [hunk 2]
```diff
@@ -1,29 +1,53 @@
 {
   "name": "@loke/http-rpc-client",
-  "version": "3.0.1",
+  "version": "3.1.0-beta.0",
```
Comment (denwilliams) [inline package.json:3]: Any reason why beta?

Just want to check it all works as a package first?

**Review by denwilliams — COMMENTED** [review]
Mostly questions

File: package.json (hunk 1 above)
Comment (smithamax) [reply package.json]: Ah yeah probably

File: package.json (hunk 2 above)
Comment (smithamax) [reply package.json:3]: Yeah was going to bump it once reviewed

**Review by denwilliams — APPROVED** [review]
I guess the best way to test it is to try out as beta

**Review by denwilliams — APPROVED** [review]

**Pull request merged**